serveradmin/common/static/js/serveradmin.js

```diff
@@ -11,38 +11,28 @@ $(document).ready(function() {
     window.spinner = {
         _running: 0,
         _spinner: $('#spinner'),
-        _toggle_css: function() {
-            if (this._spinner.css('display') === 'none') {
-                this._spinner.show();
-            }
-
-            if (this._spinner.hasClass('text-success')) {
-                this._spinner.removeClass('text-success');
-                this._spinner.addClass('text-secondary');
-
-                this._spinner.css('animation-play-state', 'paused');
-            }
-            else {
-                this._spinner.removeClass('text-secondary');
-                this._spinner.addClass('text-success');
-
-                this._spinner.css('animation-play-state', 'running');
-            }
-        },
         enable: function () {
-            this._running++;
-            if (this._running > 1) {
-                return;
-            }
+            console.debug('spinner enabled');
+            this._running = Date.now();
```

Comment:
As `running` sounds more like a boolean value I suggest to use something more like `startedAt` or the like if `running` is not used somewhere else already.

Reply:
Fair point. I will change it.

Reply:
Adjusted.